### ISSUE TITLE
Modify Zombienet to allow custom protocols inside bootnode URL.

### ISSUE THREAD

**Issue by wirednkod** [paritytech/zombienet#815]
I think we need to add [this](https://github.com/paritytech/substrate/pull/12529#issuecomment-1477500042) as a high priority on our list.

**Comment by pepoviola** [comment]
Yes, I'm working on this. We can have a quick workaround if needed @melekes ?
